tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -299,6 +305,24 @@ static string ToString(cublasStatus_t status) {
   }
 }
 
+// Decide whether to enable TENSOR_OP_MATH
+static bool TensorOpMathEnabled() {
+  static bool is_enabled = [] {
```

Comment:
What's the point of having a lambda and evaluate it right away?

Reply:
This allows the static is_enabled to be set inline.